### PR TITLE
doc(core): Update node version requirement documentation.

### PR DESCRIPTION
**Pull request by wodenx**
## Changes

- Clearly indicate that only Node 12.x is supported.

### CODE REVIEW

File: packages/bodiless-documentation/doc/README.md [hunk 1]
```diff
@@ -29,9 +29,7 @@ npm run start
 
 Then Visit `localhost:8005` in your browser.
 
-> Requires:
-> - [NodeJS](https://nodejs.org/en/download/) (v12.18.0+ preferred)
-> - [NPM](https://docs.npmjs.com/) version 6.13.1+
+> Requires [NodeJS](https://nodejs.org/en/download/) (v12.x only)
```
Comment (dewen) [inline packages/bodiless-documentation/doc/README.md:32]: v12.19.x ?

Comment (wodenx) [reply packages/bodiless-documentation/doc/README.md:32]: i think ppl can use an older 12.x version if they need to

Comment (wodenx) [reply packages/bodiless-documentation/doc/README.md:32]: not sure why they would

Comment (dewen) [reply packages/bodiless-documentation/doc/README.md:32]: right, we don't specify engine-strict

**Review by dewen — APPROVED** [review]

**Review by hvanyo — COMMENTED** [review]
Its also mentioned here examples/starter/README.md

**Review by hvanyo — APPROVED** [review]

**Pull request merged**